api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/AllergyIntoleranceTranslatorImpl.java

```diff
@@ -50,6 +61,10 @@ public AllergyIntolerance toFhirResource(Allergy omrsAllergy) {
 			}
 		}
 		allergy.setClinicalStatus(setClinicalStatus(omrsAllergy.getVoided()));
+		allergy.setPatient(patientReferenceTranslator.toFhirResource(omrsAllergy.getPatient()));
+		allergy.setRecorder(practitionerReferenceTranslator.toFhirResource(omrsAllergy.getCreator()));
+		allergy.setRecordedDate(omrsAllergy.getDateCreated());
+		allergy.setLastOccurrence(omrsAllergy.getDateLastUpdated());
```

Comment:
I don't think this is a great mapping. There are lots of reasons the `Allergy` instance might get updated without that necessarily corresponding to an occurrence of the allergy. I would suggest that we propose the `lastOccurrence` date as something added to the OpenMRS core data model. In particular, it would probably be good to add something like a "reactionDate" property to `AllergyReaction` and calculate the `lastOccurrence` from that. For now, could we eliminate this mapping?

Otherwise, this LGTM.

Reply:
@ibacher this makes sense, this  "reactionDate" would be a  better approach. Removing this for now. I'll post this on talk and see if they'll resolve to adding this property